webplugin/css/app/mck-sidebox-1.0.css

```diff
@@ -5466,12 +5466,12 @@ div#mck-rated:before{
 }
 
 .mck-send-feedback {
-	margin-top: 10px;
+	margin-top: 10px !important;
```

Comment:
please check if using !important is a good practive?
@akshatdhabi any suggestions here?

Reply:
Using `!important` is considered as a bad practice and usually should be done as a last resort as it can cause issues in CSS code down the line. Instead, we can use **CSS Specificity**  to eliminate the need of `!important` almost all the time. 

Please refer: 
https://uxengineer.com/css-specificity-avoid-important-css/

Reply:
I was already using an id on the element but the changes are still being overriden by some other styling deifned by the class of the element.

Reply:
The above-mentioned link can easily solve the problem you mentioned.